apps/web-giddh/src/app/material-ledger/components/advance-search/advance-search.component.scss

```diff
@@ -0,0 +1,95 @@
+@import "../../../../assets/css/color.scss";
+
+.clear-filter-close {
+    justify-content: flex-end;
+    align-items: center;
+}
+.clear-filter-close a {
+    color: $white;
+    font-size: 14px;
+    line-height: 2.2;
+}
+.modal-header {
+    display: flex;
+    justify-content: space-between;
+}
+.modal-header .close {
+    color: $white;
+    opacity: 1;
+    font-weight: 100;
+    text-shadow: none;
+    font-size: 30px;
+    cursor: pointer;
+}
+.collapse-icons {
+    font-size: 10px;
+    border: 1px solid $chetwode-blue;
+    border-radius: 4px;
+    padding: 2px;
+    top: 0 !important;
+}
+
+@media (min-width: 1200px) {
+    .container {
+        width: 100%;
+        max-width: 1280px;
+    }
+}
+@media only screen and (max-width: 1024px) {
+    .container {
+        width: 100%;
+    }
+}
+@media only screen and (max-width: 767px) {
+    .ledger-state-box {
+        margin-bottom: 20px;
+    }
+    .amount-total {
+        background: $whisper-light;
+    }
+    .page-count {
+        width: 100%;
+        left: 0;
+        bottom: 0;
+        position: relative;
+    }
+    .ledger-stat {
+        margin: 0px 0 0;
+    }
+    .ledger-book {
+        margin-bottom: 100px;
+    }
+    .ledger-head thead tr th.active {
+        display: block !important;
+        width: 100%;
+    }
+    .ledger-book .h-table tbody .ledger-row td {
+        position: relative;
+        padding-left: 80px !important;
+    }
+    .advance-search {
+        p {
+            line-height: 18px;
+            margin-bottom: 10px;
+        }
+        .mb-2 {
+            margin-bottom: 10px !important;
+        }
+        .mt-2 {
+            margin-top: 10px;
+        }
+        .mr-b1 {
+            margin-bottom: 0;
+        }
+    }
+}
+@media screen and (max-width: 575px) {
+    .switch-wrapper {
+        padding-right: 15px !important;
+        margin-bottom: 15px;
+    }
+}
+.mat-switch {
```

Comment:
please add this css above media queries and please do not add any css after media queries in any page.

Reply:
Done